OSS/community webapps won't build without `packages/webapps.e/`

The OSS / community version of webapps doesn't build without `packages/webapps.e/`.

Repro steps (copied from the [Teleport README](https://github.com/gravitational/teleport#web-ui)):

```sh
$ git clone git@github.com:gravitational/webapps.git
Cloning into 'webapps'...
remote: Enumerating objects: 7794, done.
remote: Counting objects: 100% (1763/1763), done.
remote: Compressing objects: 100% (813/813), done.
remote: Total 7794 (delta 1100), reused 1477 (delta 924), pack-reused 6031
Receiving objects: 100% (7794/7794), 11.99 MiB | 7.12 MiB/s, done.
Resolving deltas: 100% (4923/4923), done.
$ cd webapps
$ make build-teleport
/Library/Developer/CommandLineTools/usr/bin/make docker-build NPM_CMD=build-teleport FROM=dist/ TO=dist/;
docker build --force-rm=true --build-arg NPM_SCRIPT=build-teleport -t web-apps .
[+] Building 2.6s (15/20)
 => [internal] load build definition from Dockerfile                                                                                                                                                                     0.0s
 => => transferring dockerfile: 1.10kB                                                                                                                                                                                   0.0s
 => [internal] load .dockerignore                                                                                                                                                                                        0.0s
 => => transferring context: 139B                                                                                                                                                                                        0.0s
 => [internal] load metadata for docker.io/library/node:12-slim                                                                                                                                                          1.6s
 => [ 1/16] FROM docker.io/library/node:12-slim@sha256:41fc823270281934231f7dddae7cfbbc9c71a7c4efccbf962b9bc13b1abde2dc                                                                                                  0.0s
 => [internal] load build context                                                                                                                                                                                        0.8s
 => => transferring context: 71.60MB                                                                                                                                                                                     0.8s
 => CACHED [ 2/16] RUN apt-get update && apt-get install git -y                                                                                                                                                          0.0s
 => CACHED [ 3/16] RUN mkdir -p web-apps                                                                                                                                                                                 0.0s
 => CACHED [ 4/16] COPY yarn.lock web-apps/                                                                                                                                                                              0.0s
 => CACHED [ 5/16] COPY package.json web-apps/                                                                                                                                                                           0.0s
 => CACHED [ 6/16] COPY packages/build/ web-apps/packages/build/                                                                                                                                                         0.0s
 => CACHED [ 7/16] COPY packages/design/package.json web-apps/packages/design/                                                                                                                                           0.0s
 => CACHED [ 8/16] COPY packages/gravity/package.json web-apps/packages/gravity/                                                                                                                                         0.0s
 => CACHED [ 9/16] COPY packages/shared/package.json web-apps/packages/shared/                                                                                                                                           0.0s
 => CACHED [10/16] COPY packages/teleport/package.json web-apps/packages/teleport/                                                                                                                                       0.0s
 => ERROR [11/16] COPY README.md packages/webapps.e/gravity/package.jso[n] web-apps/packages/webapps.e/gravity/                                                                                                          0.1s
------
 > [11/16] COPY README.md packages/webapps.e/gravity/package.jso[n] web-apps/packages/webapps.e/gravity/:
------
lstat /var/lib/docker/tmp/buildkit-mount889196154/packages/webapps.e/gravity: no such file or directory
make[1]: *** [docker-build] Error 1
make: *** [build-teleport] Error 2
```

The `make build-teleport-oss` and `make build-gravity-oss` present the same error.

--- a/Dockerfile
+++ b/Dockerfile
@@ -14,8 +14,8 @@ COPY packages/shared/package.json web-apps/packages/shared/
 COPY packages/teleport/package.json web-apps/packages/teleport/
 
 # copy enterprise package.json files if present
-COPY README.md packages/webapps.e/gravity/package.jso[n] web-apps/packages/webapps.e/gravity/
-COPY README.md packages/webapps.e/teleport/package.jso[n] web-apps/packages/webapps.e/teleport/
+COPY README.md packages/webapps.e/gravit[y]/package.json web-apps/packages/webapps.e/gravity/
+COPY README.md packages/webapps.e/telepor[t]/package.json web-apps/packages/webapps.e/teleport/
 
 # download and install npm dependencies
 WORKDIR web-apps

